src/index.html

```diff
@@ -10,7 +10,44 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Search bar airbnb</h1>
+  <body class="body">
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="get"
+      class="firstForm"
```

Comment:
You do not need different classes for forms
```suggestion
      class="form"
```